### PR TITLE
faq goodies text removed

### PR DESCRIPTION
**Pull request by anistark**
This year ticket price does not include any goodies or t-shirt.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ This site is built with [Jekyll](https://jekyllrb.com/). For a basic introductio
 * Fork the repository and clone to your machine by using `git clone https://github.com/<username>/inpycon2020`.
 * `cd` to the repo and run `bundle install` to install all dependencies.
 * Run `bundle exec jekyll serve`.
-* Visit the local development server at `http://localhost:4000/2020`.
+* Visit the local development server at http://127.0.0.1:4000/2020/.
```
Comment (palnabarun) [inline README.md]: Shouldn't it be a separate commit?

File: faq/index.md [hunk 2]
```diff
@@ -53,8 +53,6 @@ A full conference ticket for two days includes access to all the amazing keynote
 
 You will also get access to all the open space discussions and other sessions during the conference. The conference ticket holders will also be eligible to join the Devsprints.
 
-The ticket also includes all the conference goodies and swags for the attendees.
```
Comment (vipulgupta2048) [inline faq/index.md:56]: Thanks @anistark looks good. 
Would it be better to have a separate FAQ clearing this bit out as suggested by @inovizz on #68 
Because even though we want to say the ticket price doesn't include swag. We don't as off yet mention it anywhere specifically. I think we can add the same in this PR. If not, LGTM

**Review by vipulgupta2048 — APPROVED** [review]

**Comment by palnabarun** [comment]
@anistark -- Can you please reword your commit by following https://chris.beams.io/posts/git-commit/?

File: README.md (hunk 1 above)
Comment (anistark) [reply README.md]: Okay. Removing.

File: faq/index.md (hunk 2 above)
Comment (anistark) [reply faq/index.md:56]: This is quick push. We don't want anymore folks to look at this and raise questions. I certainly a detailed FAQ would help once reviewed,

Comment (Sc0rpi0n101) [reply faq/index.md:56]: I can add a separate ques for swags in #71 as I'll also have to update that to remove the swags line

**Comment by anistark** [comment]
> @anistark -- Can you please reword your commit by following https://chris.beams.io/posts/git-commit/?

Done. @palnabarun

**Review by palnabarun — APPROVED** [review]
